docs/sources/setup/configuration.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: '/configuration'
+title: 'Configuration'
+menuTitle: Configuration
+description: Configuration of the GitHub data source plugin
+aliases:
+  - github
+keywords:
+  - data source
+  - github
+  - github repository
+  - API
+labels:
+  products:
+    - oss
+    - enterprise
+    - cloud
+weight: 102
+---
+
+# Configuration
+
+You will need a **GitHub Access Token** to use the plugin. Below is a table that indicates what minimum requirements must be matched before the plugin can be used.
```

Comment:
I think we shouldn't use bold for emphasis here.

Can we rewrite the table to be a list of explaining each thing you need. As a new user, I'm only interested in what I need to do to configure the plugin and I don't know what "Default Organization" not being required means.

```suggestion
You will need a GitHub Access Token to use the plugin.
```

Reply:
Also it seems that GitHub calls them "personal access tokens": https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens

We should use the same terminology if we can.

Reply:
fixed now